charts/kamaji/Chart.yaml

```diff
@@ -5,7 +5,7 @@ description: Kamaji is a tool aimed to build and operate a Managed Kubernetes Se
   hundreds of Kubernetes clusters as a hyper-scaler.
 home: https://github.com/clastix/kamaji
 icon: https://github.com/clastix/kamaji/raw/master/assets/kamaji-logo.png
-kubeVersion: 1.21 - 1.25
+kubeVersion: 1.21.0-0 - 1.25.0-0
```

Comment:
The e2e hasn't been triggered, and it's ok.

May I ask you to test on a Kind cluster, just to be sure this is not breaking?

Reply:
Yep. Doing that as well